{{cookiecutter.project_name}}/databricks-config/README.md

```diff
@@ -87,6 +87,21 @@ principal corresponding to a particular environment has permissions to read the
 * `SELECT` permission for the input table.
 * `MODIFY` permission for the output table if it pre-dates your job.
 
+### Setting up model validation
+The model validation stack focuses on building a plug-and-play stack component for continuous deployment (CD) of models 
+in staging and prod. Its central purpose is to evaluate a registered model and validate its quality before moving it to 
+Production stage in model registry.
```

Comment:
"Moving it to Production stage" may be limiting - right now we do this for Staging as well. I'd say keep this more flexible: "validate its quality before promoting it to Production/Staging stage in the model registry."

or more concise: "validate its quality before deploying the model to Production/Staging."